corehq/apps/hqadmin/management/commands/clone_domain.py

```diff
@@ -148,7 +148,7 @@ def copy_applications(self, old_domain, new_domain, report_map):
         apps = get_apps_in_domain(old_domain)
         for app in apps:
             for module in app.modules:
-                if module.module_type == ReportModule.module_type:
```

Comment:
Turns out ReportModule.module_type is a `jsonobject.properties.StringProperty` object